source/_patterns/00-atoms/icons/icons.mustache

```diff
@@ -0,0 +1,259 @@
+<h2>Icons</h2>
```

Comment:
You can wrap all of this in a `<div class="pad-double">` wrapper and that'll help with the padding.